src/css/base/common.css

```diff
@@ -15,7 +15,7 @@ ol {
 }
 
 body {
-  font-family: 'DM', sans-regular;
+  font-family: 'DM Sans', sans-regular;
```

Comment:
This file is shared, it cannot be modified or committed! The font has already been added to the main branch.